apis/v1alpha2/backendtlspolicy_types.go

```diff
@@ -0,0 +1,156 @@
+
+// BackendTLSPolicy provides a way to publish TLS configuration
+// that enables a gateway client to connect to a backend pod.
+type BackendTLSPolicy struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec defines the desired state of BackendTLSPolicy.
+	Spec BackendTLSPolicySpec `json:"spec"`
+
+	// Status defines the current state of BackendTLSPolicy.
+	Status PolicyStatus `json:"status,omitempty"`
+}
+
+// +kubebuilder:object:root=true
+// BackendTLSPolicyList contains a list of TCPRoute
+type BackendTLSPolicyList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []BackendTLSPolicy `json:"items"`
+}
+
+// BackendTLSPolicySpec defines the desired state of
+// BackendTLSPolicy.
+// Note: there is no Override or Default policy configuration.
+//
+// Support: Extended
+type BackendTLSPolicySpec struct {
+	// TargetRef identifies an API object to apply policy to.
+	// Services are the only valid API target references.
+	// Note that this config applies to the entire referenced resource
+	// by default, but this default may change in the future to provide
+	// a more granular application of the policy.
+	TargetRef PolicyTargetReference `json:"targetRef"`
+
+	// TLS contains backend TLS policy configuration.
+	TLS *BackendTLSPolicyConfig `json:"tls"`
+}
+
+// BackendTLSPolicyConfig contains backend TLS policy configuration.
+// +kubebuilder:validation:XValidation:message="must not contain both CertRefs and StandardCerts",rule="(has(self.certRefs) && size(self.certRefs) > 0 && has(self.standardCerts) && self.standardCerts != ”)"
+// +kubebuilder:validation:XValidation:message="must specify either CertRefs or StandardCerts",rule="!(has(self.certRefs) && size(self.certRefs) > 0 || has(self.standardCerts) && self.standardCerts != ”)"
+type BackendTLSPolicyConfig struct {
+	// CertRefs contains one or more references to
+	// Kubernetes objects that contain PEM-encoded TLS certificates,
+	// which are used to establish a TLS handshake between the gateway
+	// and backend pod.
+	//
+	// If CertRefs is empty or unspecified, then StandardCerts must
+	// be specified.  Only one of CertRefs or StandardCerts may be
+	// specified, not both.
+	//
+	// If CertRefs is empty or unspecified, then system trusted
+	// certificates should be used. If there are none, or the
+	// implementation doesn't define system trusted certificates,
+	// then a TLS connection must fail.
+	//
+	// References to a resource in a different namespace are
+	// invalid.
+	//
+	// A single CertRef to a Kubernetes ConfigMap kind has "Core"
+	// support.  Implementations MAY choose to support attaching
+	// multiple certificates to a backend, but this behavior is
+	// implementation-specific.  Also implementation-specific is
+	// a CertRef of other object kinds, e.g. Secret.
+	//
+	// Support: Core - An optional single reference to a Kubernetes
+	// ConfigMap.
```

Comment:
Do we have a standard key that we're expecting the CA Cert to be set on?
```suggestion
	// Support: Core - A single reference to a Kubernetes ConfigMap.
```

Reply:
Good point, updated.